### PR TITLE
New dev cycle

### PR DESCRIPTION
**Pull request by Tapchicoma**
- Updated versions.
- Update Gradle to `5.1.1` version in samples (blocker: https://github.com/JetBrains/kotlin-native/issues/2916) and to `5.4` in plugin project.
- Simplify plugin setup in samples.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -4,16 +4,13 @@ buildscript {
...
-        classpath("org.jlleitschuh.gradle:ktlint-gradle:+")
-    }
 }
 
 plugins {
     kotlin("jvm") version SamplesVersions.kotlin apply false
     id("com.android.application") version SamplesVersions.androidPlugin apply false
-    id("konan") version SamplesVersions.kotlin apply false
+    id("org.jetbrains.kotlin.konan") version SamplesVersions.kotlin apply false
+    id("org.jlleitschuh.gradle.ktlint")
```
Comment (JLLeitschuh) [inline build.gradle.kts]: ※\\(^o^)/※

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-bin.zip
```
Comment (JLLeitschuh) [inline gradle/wrapper/gradle-wrapper.properties]: Should be `ALL` distribution right?

**Comment by JLLeitschuh** [comment]
Does this break backward compatibility with the android/kotlin plugins?

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (Tapchicoma) [reply gradle/wrapper/gradle-wrapper.properties]: This wrapper is used for samples project. Plugin project (that is included into samples project as composite build) uses `ALL` distribution.

**Comment by Tapchicoma** [comment]
> Does this break backward compatibility with the android/kotlin plugins?

Should not.

**Pull request merged**